### PR TITLE
Add message in status bar if user.name and/or user.email not configured

### PR DESCRIPTION
**Pull request by vparfonov**
### What does this PR do?

Add message in status bar if user.name and/or user.email not configured

che#14617

### What issues does this PR fix or reference?
eclipse/che#14617

### CODE REVIEW

File: extensions/eclipse-che-theia-git-provisioner/package.json [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
+      "\\.(css|less)$": "<rootDir>/tests/mock.js"
+    },
+    "moduleFileExtensions": [
+      "js",
+      "ts"
+    ]
   }
-  
\ No newline at end of file
+}
```
Comment (evidolob) [inline extensions/eclipse-che-theia-git-provisioner/package.json]: New line

Comment (vparfonov) [reply extensions/eclipse-che-theia-git-provisioner/package.json]: fixed

**Comment by vparfonov** [comment]
crw-ci-test

**Comment by musienko-maxim** [comment]
crw-ci-test

**Comment by musienko-maxim** [comment]
We have unexpected problems with slave on the Up Shift, so i used devfile from the this PR and relaunch the tests locally. All tests passed.

**Comment by vparfonov** [comment]
> We have unexpected problems with slave on the Up Shift, so i used devfile from the this PR and relaunch the tests locally. All tests passed.

Thanks

**Pull request merged**